plugins/tiddlywiki/jasmine/files/reporter.js

```diff
@@ -176,7 +176,7 @@
       var results = runner.results();
       var specs = runner.specs();
       var msg = '';
-      msg += specs.length + ' test' + ((specs.length === 1) ? '' : 's') + ', ';
+      msg += specs.length + ' template.html' + ((specs.length === 1) ? '' : 's') + ', ';
```

Comment:
What's the idea behind this change?

Reply:
I guess this was a mistakenly commited file after refactoring my template :D shouldn't be here